lua/nordic/groups/integrations.lua

```diff
@@ -97,39 +97,82 @@ function M.get_groups()
     G.MiniStatuslineFileinfo = { bg = LC.normal.b.bg, fg = LC.normal.b.fg }
     G.MiniStatuslineInactive = { bg = LC.inactive.a.bg, fg = LC.inactive.a.fg, bold = LC.inactive.a.gui == 'bold' }
 
+    -- Tree Groups
+    G.TreeNormal = { fg = C.fg, bg = C.bg }
+    G.TreeNormalNC = { fg = C.fg, bg = C.bg }
+    G.TreeRootName = { fg = C.fg, bold = true }
+    G.TreeFileIcon = { fg = C.blue2 }
+    G.TreeFileNameOpened = { fg = C.fg }
+    G.TreeSpecialFile = { fg = C.magenta.bright }
+    G.TreeGitConflict = { fg = C.red.base }
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
+    G.TreeTitleBar = { link = "WinBar"}
+    G.TreeFloatBorder = { fg = C.border_fg, bg = C.bg_float }
```

Comment:
I think we mislead you on this one:
```suggestion
    G.TreeFloatBorder = { link = "FloatBorder" }
```